PocketKit/Sources/PocketKit/Tags/TagsFilterViewModel.swift

```diff
@@ -21,7 +21,8 @@ class TagsFilterViewModel: ObservableObject {
     }
 
     func getAllTags() -> [TagType] {
-        arrangeTags(with: fetchedTags?.compactMap({ $0.name }) ?? [])
+        // Finding unique elements using set
+        arrangeTags(with: Array(Set(fetchedTags?.compactMap({ $0.name }) ?? [])))
```

Comment:
I had to do this, because at times the FilterTagsView would contain duplicate tags. Should be fixed in https://github.com/Pocket/pocket-ios/pull/568